.coveragerc

```diff
@@ -77,6 +77,10 @@ omit =
     homeassistant/components/daikin/const.py
     homeassistant/components/*/daikin.py
 
+    homeassistant/components/depict/__init__.py
+    homeassistant/components/light/depict.py
```

Comment:
Once all files have been moved over to `homeassistant/components/depict/`, this can be changed to a single line: `homeassistant/components/depict/*`

Please make sure to rebase since there is a conflict in this file.